test/augmentation/test_augmentation.py

```diff
@@ -4036,6 +4037,7 @@ def test_smoke(self, device, dtype):
         out = aug(img)
         assert out.shape == (2, 3, 4, 5)
 
+    @pytest.mark.skipif(torch_version() == '2.0.0', reason='Not working on 2.0')
```

Comment:
Open later issues or directly to pytorch

Reply:
It seems to be already fixed on the nightly version (2.1) 😄

Reply:
then why not supporting 2.1 directly ? in the past we had similar issues and we ended up supporting versions not ended with patch version == 0

Reply:
since they just release the 2.0, I did not expect the 2.1 to be out already this month

Reply:
ok, then let's merge this as it is